ApiReview/API.Polly.RateLimiting/NoDocs/Polly.RateLimiting/RateLimiterStrategyOptions.cs

```diff
@@ -0,0 +1,18 @@
+// Assembly 'Polly.RateLimiting'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Threading.RateLimiting;
+using System.Threading.Tasks;
+
+namespace Polly.RateLimiting;
+
+public class RateLimiterStrategyOptions : ResilienceStrategyOptions
+{
+    [Required]
+    public ConcurrencyLimiterOptions DefaultRateLimiterOptions { get; set; }
+    public Func<OnRateLimiterRejectedArguments, ValueTask>? OnRejected { get; set; }
+    public ResilienceRateLimiter? RateLimiter { get; set; }
```

Comment:
```suggestion
   public Func<RateLimiterArguments, ValueTask<RateLimitLease>> RateLimiter { get; set; }
```

And get rid of `ResilienceRateLimiter`. We would have to solve the problem with resource management i.e. disposing unused rate limiters after pipeline is reloaded or disposed.

Reply:
Addressed by #1509